### PR TITLE
Add torus font

### PR DESCRIPTION
**Pull request by peppy**
Replaces existing usages of Aquatico with torus. Design fixes will come in follow-up PRs (for OWC 2020 series design).

### CODE REVIEW

File: osu.Game/Graphics/OsuFont.cs [hunk 1]
```diff
@@ -65,16 +70,7 @@ public static string GetWeightString(Typeface typeface, FontWeight weight)
         /// <param name="family">The family string.</param>
         /// <param name="weight">The <see cref="FontWeight"/>.</param>
         /// <returns>The string representation of <paramref name="weight"/> in the specified <paramref name="family"/>.</returns>
-        public static string GetWeightString(string family, FontWeight weight)
-        {
-            string weightString = weight.ToString();
-
-            // Only exo has an explicit "regular" weight, other fonts do not
-            if (family != GetFamilyString(Typeface.Exo) && weight == FontWeight.Regular)
```
Comment (smoogipoo) [inline osu.Game/Graphics/OsuFont.cs]: Have you checked that this isn’t required still? I’m fairly certain this breaks Venera.

**Review by smoogipoo — CHANGES_REQUESTED** [review]

File: osu.Game/Graphics/OsuFont.cs (hunk 1 above)
Comment (peppy) [reply osu.Game/Graphics/OsuFont.cs]: aha, didn't see that case, looks bad. we probably want to rename this and remove "medium" altogether (or standardise to regular).

Comment (peppy) [reply osu.Game/Graphics/OsuFont.cs]: Turns out all the naming for venera was wrong (and one instance existed without a weight suffix). Should be better aligned now.

File: osu.Game/Graphics/OsuFont.cs [hunk 2]
```diff
@@ -102,15 +98,39 @@ public enum Typeface
...
     public enum FontWeight
     {
-        Light,
-        Regular,
-        Medium,
-        SemiBold,
-        Bold,
-        Black
+        /// <summary>
+        /// equivalent to weight 300
```
Comment (smoogipoo) [inline osu.Game/Graphics/OsuFont.cs]:
```suggestion
        /// Equivalent to weight 300.
```

File: osu.Game/Graphics/OsuFont.cs [hunk 3]
```diff
@@ -102,15 +98,39 @@ public enum Typeface
...
+        /// </summary>
+        Light = 300,
+
+        /// <summary>
+        /// equivalent to weight 400
```
Comment (smoogipoo) [inline osu.Game/Graphics/OsuFont.cs]:
```suggestion
        /// Equivalent to weight 400.
```

File: osu.Game/Graphics/OsuFont.cs [hunk 4]
```diff
@@ -102,15 +98,39 @@ public enum Typeface
...
+        /// </summary>
+        Regular = 400,
+
+        /// <summary>
+        /// equivalent to weight 500
```
Comment (smoogipoo) [inline osu.Game/Graphics/OsuFont.cs]:
```suggestion
        /// Equivalent to weight 500.
```

File: osu.Game/Graphics/OsuFont.cs [hunk 5]
```diff
@@ -102,15 +98,39 @@ public enum Typeface
...
+        /// </summary>
+        Medium = 500,
+
+        /// <summary>
+        /// equivalent to weight 600
```
Comment (smoogipoo) [inline osu.Game/Graphics/OsuFont.cs]:
```suggestion
        /// Equivalent to weight 600.
```

File: osu.Game/Graphics/OsuFont.cs [hunk 6]
```diff
@@ -102,15 +98,39 @@ public enum Typeface
...
+        /// </summary>
+        SemiBold = 600,
+
+        /// <summary>
+        /// equivalent to weight 700
```
Comment (smoogipoo) [inline osu.Game/Graphics/OsuFont.cs]:
```suggestion
        /// Equivalent to weight 700.
```

File: osu.Game/Graphics/OsuFont.cs [hunk 7]
```diff
@@ -102,15 +98,39 @@ public enum Typeface
...
+        /// </summary>
+        Bold = 700,
+
+        /// <summary>
+        /// equivalent to weight 900
```
Comment (smoogipoo) [inline osu.Game/Graphics/OsuFont.cs]:
```suggestion
        /// Equivalent to weight 900.
```

**Review by smoogipoo — CHANGES_REQUESTED** [review]

**Pull request merged**